client/src/components/Login.tsx

```diff
@@ -39,80 +32,97 @@ function Login(props: { handleChange: Function }) {
   };
   const validateBlurPassword = (regexPattern: RegExp) => {
     if (!regexPattern.test(password)) {
-      setPasswordError("Ensure password is 8 characters or long");
+      setPasswordError("Ensure password is 8 characters or longer");
     } else {
       setPasswordError("");
     }
   };
 
   // handle function for submitting username and password
-  const handleSubmit = (e: React.ChangeEvent<{}>) => {
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault(); // no refresh; default
 
     // TODO api calls here after submit
   };
 
   return (
-    <Paper style={paperStyle} data-testid="LogInTab">
-      <Grid container justifyContent="center">
-        <Grid item>
-          <Avatar style={avatarStyle}>
-            <LockOutlined fontSize="large" />
-          </Avatar>
-        </Grid>
-      </Grid>
-
-      <Box textAlign="center">
-        <h1>Log In</h1>
-      </Box>
-      <form noValidate autoComplete="off" onSubmit={handleSubmit}>
+    <Box
+      data-testid="LogInTab"
+      sx={{
+        mx: 2,
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+      }}
+      overflow="auto"
+      maxHeight={height - 200 > 0 ? height - 200 : 0}
```

Comment:
It's bad practice to calculate the height since it happens on each render. Can we make it full height, then have a padding, or margin?

Reply:
I believe the height is only calculated in the event that the window is resized. 

I have a hook in window.tsx that only calculates the height when an resize window eventListener is triggered.